src/mugs.js

```diff
@@ -480,7 +480,7 @@ define([
             return this.parentMug && this.parentMug.isInRepeat();
         },
         supportsRichText: function() {
-            return this.options.richText && this.form.useRichText !== false;
+            return this.options.richText && this.form.noRichText === false;
```

Comment:
although, having a boolean flag use different semantics for `false` and `undefined` would be a bad idea anyway. i'll update this.